tests/parser/test_preprocessor.py

```diff
@@ -0,0 +1,19 @@
+"""Unit tests for preprocessors."""
+
+from fonduer.parser.preprocessors.hocr_doc_preprocessor import HOCRDocPreprocessor
+
+
+def test_hocrpreprocessor():
+    """Test hOCRDocPreprocessor."""
+    path = "tests/data/hocr_simple/md.hocr"
+    preprocessor = HOCRDocPreprocessor(path=path)
+    doc = next(iter(preprocessor))
+    assert doc.name == "md"
```

Comment:
Might be nice to have some more specific spot checking of some words or something more detailed besides just the title?

Reply:
Resolved by d0c96fb02d2708228129535da3db5ccc90f9ef01.